test/src/test/groovy/org/openremote/test/protocol/http/HttpClientProtocolTest.groovy

```diff
@@ -549,4 +563,136 @@ class HttpClientProtocolTest extends Specification implements ManagerContainerTr
             HTTPProtocol.client.set(null)
         }
     }
+
+    def "Check HTTP client dynamic time feature"() {
```

Comment:
This test doesn't cover the different scenarios:

* `{$time}` Only
* time with +/-offset
* time with no offset but a format request